src/index.html

```diff
@@ -11,12 +11,107 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images\logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="apple is-active"
```

Comment:
Class names should represent the meaning of the content, not the styles. The class 'is-active' is a state indicator, which is fine, but 'apple' seems to indicate style or tag, which should be avoided.